### DISCUSSION TITLE
What's the expected FRESH install time for a Hi Speed Network connection?

### DISCUSSION THREAD

**Discussion by ashrocket** [Homebrew/discussions#3090] [Getting started]
Can folks comment on how long it takes their initial install to run?
We are seeing 45 minutes on a highspeed network and are not sure what the source of the issue might be (but maybe it's the full clone of homebrew/core that's required after Github deprecated shallow clones)

https://github.com/Homebrew/discussions/discussions/226

**Comment by SMillerDev** [comment]
It's probably the commandline tools. Apple downloads are notoriously slow.
